cmd/helm/completion.go

```diff
@@ -52,7 +52,7 @@ func newCompletionCmd(out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "completion SHELL",
-		Short: "generate autocompletions script for the specified shell (bash or zsh)",
+		Short: "generate autocompletions script for the specified shell (bash, zsh or fish)",
```

Comment:
I think the long description also needs to include 'fish'

Reply:
Agreed, tottaly missed that one

Reply:
76c63b819325c72bfb4f6f7460ff52a83bfb9c10 should fix it